public/angular/bower.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "fc-app",
+  "description": "",
+  "main": "",
+  "authors": [
+    "<none>"
+  ],
+  "license": "MIT",
+  "homepage": "https://github.com/FastingCrusade/website",
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests"
```

Comment:
This is something bower made on init.  I'm not actually sure what exactly it is ignoring, but if we do eventually do actual tests they should certainly be committed.